steamPackages.steam: 1.0.0.61 -> 1.0.0.68

jonringer opened the pull request:
###### Motivation for this change
was trying to hack around and make NixOS work with the latest proton runtime... but just settled for updating the package today. And doing some minor cleanups

udev rules are no longer shipped with steam, so I removed the related logic
fixed the desktop file inside the unwrapped steam package, instead of the wrapper

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

Commit:
steam-fhsenv: normalize ldPath

Commit:
steamPackages.steam: 1.0.0.61 -> 1.0.0.68

Commit:
steam: fix desktop link

andir commented on pkgs/games/steam/fhsenv.nix:
```diff
@@ -43,12 +44,14 @@ let
 
-  ldPath = map (x: "/steamrt/${steam-runtime-wrapped.arch}/" + x) steam-runtime-wrapped.libs
-           ++ lib.optionals (steam-runtime-wrapped-i686 != null) (map (x: "/steamrt/${steam-runtime-wrapped-i686.arch}/" + x) steam-runtime-wrapped-i686.libs);
+  ldPath = lib.optionals stdenv.is64bit [ "/lib64"]
```
Please add a whitespace after `"lib64"`. Just to be consistent. :)

jonringer replied:
ah, yes

mweinelt reviewed (commented):
Among us ✔️  (upgraded to Proton 5.0-10)
Stellaris ✔️ 

On every start of Steam I now get 🤔 

```
cp: cannot create regular file '/home/hexa/.local/share/Steam/bootstrap.tar.xz': Permission denied
```

After deleting it steam starts okay.

```
.r--r--r--  2.9M hexa 21 Dez  1:55  .local/share/Steam/bootstrap.tar.xz
```

Commit:
steam: fix readonly boostrap.tar.xz

steam writes it with 444 permissions, which makes
simple deletions fails, and steam will crash

jonringer commented:
> After deleting it steam starts okay.

I forgot that I also did this :)

mweinelt commented:
Looking good, WFM.

jonringer commented:
able to play csgo, CK III, ONI fine
